journal/journal.go

```diff
@@ -137,6 +137,17 @@ func (page JournalsPage) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a)
 }
 
+type ClientsTelemetry struct {
```

Comment:
Use singular `ClientTelemetry` instead, since this is a telemetry for a single client.